lib/aadutils.js

```diff
@@ -22,6 +22,7 @@
  */
 'use strict';
 
+const base64url = require('base64url');
 const crypto = require('crypto');
```

Comment:
@polita @RandalliLama @aashiman @tushargupta51 No, crypto is the official one from nodejs (https://nodejs.org/api/crypto.html#crypto_crypto)